airflow-core/src/airflow/ui/public/i18n/locales/de/dag.json

```diff
@@ -25,12 +25,17 @@
   "logs": {
     "allLevels": "Alle Protokoll-Stufen",
     "allSources": "Alle Quellen",
+    "critical": "KRITISCH",
```

Comment:
FYI. We decided not to translate those in PL  as log level is really "technical" term and the English levels are popular.

But of course every language's mileage varies

Reply:
Pretty Agree. This is why I left them as English in PL translation

Reply:
Actuallly (@pierrejeambrun @bbovenzi ) -> Maybe that is a good reason to NOT provide the option to translate log levels at all and remove the keys ? That sounds like a universal problem in every language.

Reply:
Oh, yeah, did not see this - so a translation in this case really is "just" adding inconsistencies. Still I'd like to have the "ability" to translate but then probably the string also need to be replaces in all utilitities rendering the logs consistently.